quickstart/speech-translation/csharp-uwp/helloworld.sln

```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
include the temporary key pfx that visual studio generates.
make sure you can build and run this sample from a fresh Git clone of your branch.

Reply:
Working on this now.